lib/LaTeXML/Common/Error.pm

```diff
@@ -357,11 +357,9 @@ sub MergeStatus {
     if ($type eq 'undefined' or $type eq 'missing') {
       my $table = $$external_status{$type};
       foreach my $subtype(keys %$table) {
-        $$status{$type}{$subtype} ||= 0;
         $$status{$type}{$subtype} += $$table{$subtype};
       }
     } else {
-      $$status{$type} ||= 0;
```

Comment:
I have gotten used to being extra cautious from my Ruby days, let me know if you want these guards back @brucemiller

Reply:
works just fine without them naturally